styles/style.css

```diff
@@ -6,3 +6,119 @@ Green: #00B172
 White: #FFF
 
 */
+body {
+    margin: 0;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+/* NAV BAR*/
+#nav-main {
+    display: flex;
+    position: fixed;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0px 20px;
+    width: calc(100% - 40px);
+    top: 0;
+    align-content: center;
+    background-color: #FFF;
+}
+#nav-ul {
+    display: flex;
+    list-style-type: none;
+    justify-content: space-between;
+    font-size: 120%;
+}
+nav img {
+    height: 50px;
+    padding: 10px;
+}
+.pad {
+    padding: 0 15px;
+    font-size: 120%;
+}
+
+/* LANDING IMAGE */
+#landing-image {
+    background-image: url("/images/landing.jpg");
```

Comment:
You should route the image like this:
```css
    background-image: url("./../images/landing.jpg");
``` 
Otherwise it won't be relative, meaning I couldn't see the image in my computer